docs/json/radarr/cf/anime-bd-tier-05-remuxes.json

```
@@ -57,7 +57,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(PMR)\\b.*(Remux)"
+        "value": "\\b(PMR)\\b.*(Remux)|(Remux).*\\b(PMR)\\b"
```

```suggestion
        "value": "^(?=.*\\b(PMR)\\b)(?=.*\\b(Remux)\\b)"
```
I would suggest this, so it doesn't matter in what order they appear
https://regex101.com/r/NEl9sk/1